InnerEye/ML/deep_learning_config.py

```diff
@@ -230,6 +230,13 @@ class WorkflowParams(param.Parameterized):
     monitoring_interval_seconds: int = param.Integer(0, doc="Seconds delay between logging GPU/CPU resource "
                                                             "statistics. If 0 or less, do not log any resource "
                                                             "statistics.")
+    regression_test_folder: Optional[Path] = \
```

Comment:
It's odd to have test related stuff in the non-test code. I'm guessing that's essential?

Reply:
It is essential here, yes.